includes/tracks/events/class-wc-extensions-tracking.php

```diff
@@ -76,4 +77,21 @@ public function track_helper_disconnected() {
 	public function track_helper_subscriptions_refresh() {
 		WC_Tracks::record_event( 'extensions_subscriptions_update' );
 	}
+
+	/**
+	 * Send a Tracks event when addon is installed via the Extensions page.
+	 *
+	 * @param string $addon_id Addon slug.
+	 */
+	public function track_addon_install( $addon_id = '' ) {
+		if ( empty( $addon_id ) ) {
+			return;
+		}
+
+		$properties = array( 'context' => 'extensions' );
+
+		if ( 'woocommerce-payments' === $addon_id ) {
+			WC_Tracks::record_event( 'woocommerce_payments_install', $properties );
+		}
```

Comment:
Not critical, but switch/case would look better here IMO:
```suggestion
		switch ( $addon_id ) {
		    case 'woocommerce-payments':
		        WC_Tracks::record_event( 'woocommerce_payments_install', array( 'context' => 'extensions' ) );
		        break;
		}
```

Reply:
I thought about this but didn't want to go that route until tracking for other addons were added here.

Reply:
Ok, no problem, any solution would work for start :)